crates/common_enums/src/enums.rs

```diff
@@ -773,6 +773,375 @@ impl Currency {
             | Self::ZAR => false,
         }
     }
+
+    pub fn to_currency_symbol(&self) -> CurrencySymbol {
```

Comment:
can we remove this or move this to js file, as this is not a feature that backend wants